npm/npm_import.bzl

```diff
@@ -174,6 +175,12 @@ def npm_import(
             the `direct` attribute of the generated npm_link_package.
         run_lifecycle_hooks: If true, runs `preinstall`, `install` and `postinstall` lifecycle hooks declared in this
             package.
+        lifecycle_hooks_env: Environment variables applied to the lifecycle hooks scripts.
```

Comment:
Applied to which lifecycle hooks? Is it just "pnpm lifecycle hooks"? Just to be more clear.

Probably drop the word "scripts"?

Reply:
Got it, I update the doc based on the `run_lifecycle_hooks`.